tracing-forest/src/layer/mod.rs

```diff
@@ -356,9 +360,9 @@ where
 /// ```
 /// Produces the the output:
 /// ```log
-/// INFO     ＿ [info]: Hello, world!
+/// INFO     ｉ [info]: Hello, world!
 /// INFO     my_span [ 26.0µs | 100.000% ]
-/// INFO     ┕━ ＿ [info]: Relevant information
+/// INFO     ┕━ ｉ [info]: Relevant information
```

Comment:
I still say this should be just two spaces.

Reply:
I guess it doesn't really matter that much, but I'm curious why `ｉ` isn't as good at two spaces? I think it aligns with info better and is equally unobtrusive.